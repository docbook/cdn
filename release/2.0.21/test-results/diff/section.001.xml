<html xmlns:deltaxml="http://www.deltaxml.com/ns/well-formed-delta-v1" xmlns="http://www.w3.org/1999/xhtml" xmlns:dxx="http://www.deltaxml.com/ns/xml-namespaced-attribute" xmlns:dxa="http://www.deltaxml.com/ns/non-namespaced-attribute" deltaxml:deltaV2="A!=B" deltaxml:version="2.0" deltaxml:content-type="full-context">
   <head deltaxml:deltaV2="A=B">
      <link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css"/>
      <link rel="stylesheet" type="text/css" href="../../resources/css/default.css"/>
      <link href="../../resources/css/prism.css" rel="stylesheet" type="text/css"/>
      <meta charset="utf-8"/>
      <meta name="generator" content="DocBook XSL 2.0 Stylesheets Vxxx"/>
      <script type="text/javascript" src="../../resources/js/dbmodnizr.js"/>
      <title>Article wrapper</title>
   </head>
   <body deltaxml:deltaV2="A!=B">
      <article deltaxml:deltaV2="A!=B" id="R.1" class="article">
         <header deltaxml:deltaV2="A=B" class="article-titlepage">
            <h2>Article wrapper</h2>
         </header>
         <div deltaxml:deltaV2="A=B" class="lists-of-titles">
            <div class="toc">
               <p>
                  <b>Table of Contents</b>
               </p>
               <ul class="toc">
                  <li>
                     <span>
                        <a href="#c2s1">1. Section 1</a>
                     </span>
                     <ul class="toc">
                        <li>
                           <span>
                              <a href="#c2s1s1">1.1. Section 1.1</a>
                           </span>
                        </li>
                        <li>
                           <span>
                              <a href="#c2s1s2">1.2. Section 1.2</a>
                           </span>
                        </li>
                        <li>
                           <span>
                              <a href="#c2s1s3">1.3. Section 1.3</a>
                           </span>
                        </li>
                     </ul>
                  </li>
               </ul>
            </div>
         </div>
         <p deltaxml:deltaV2="A">See <a href="#c2s1s1s2" title="Section 1.1.2">Section 1.1.2, “Section 1.1.2”</a>.</p><div deltaxml:deltaV2="B" class="content">
            <p>See <a href="#c2s1s1s2" title="Section 1.1.2">Section 1.1.2, “Section 1.1.2”</a>.</p>
            <section id="c2s1" class="section">
               <div class="section-titlepage">
                  <h2>1. Section 1</h2>
               </div>
               <div class="content">
                  <p>blah.</p>
                  <section id="c2s1s1" class="section">
                     <div class="section-titlepage">
                        <h3>1.1. Section 1.1</h3>
                     </div>
                     <div class="content">
                        <p>blah.</p>
                        <section id="c2s1s1s1" class="section">
                           <div class="section-titlepage">
                              <h4>1.1.1. Section 1.1.1</h4>
                           </div>
                           <div class="content">
                              <p>blah.</p>
                              <section id="c2s1s1s1s1" class="section">
                                 <div class="section-titlepage">
                                    <h5>1.1.1.1. Section 1.1.1.1</h5>
                                 </div>
                                 <div class="content">
                                    <p>blah.</p>
                                    <section id="c2s1s1s1s1s1" class="section">
                                       <div class="section-titlepage">
                                          <h6>1.1.1.1.1. Section 1.1.1.1.1</h6>
                                       </div>
                                       <div class="content">
                                          <p>blah.</p>
                                       </div>
                                    </section>
                                    <section id="c2s1s1s1s1s2" class="section">
                                       <div class="section-titlepage">
                                          <h6>1.1.1.1.2. Section 1.1.1.1.2</h6>
                                       </div>
                                       <div class="content">
                                          <p>blah.</p>
                                       </div>
                                    </section>
                                    <section id="c2s1s1s1s1s3" class="section">
                                       <div class="section-titlepage">
                                          <h6>1.1.1.1.3. Section 1.1.1.1.3</h6>
                                       </div>
                                       <div class="content">
                                          <p>blah.</p>
                                       </div>
                                    </section>
                                 </div>
                              </section>
                              <section id="c2s1s1s1s2" class="section">
                                 <div class="section-titlepage">
                                    <h5>1.1.1.2. Section 1.1.1.2</h5>
                                 </div>
                                 <div class="content">
                                    <p>blah.</p>
                                 </div>
                              </section>
                              <section id="c2s1s1s1s3" class="section">
                                 <div class="section-titlepage">
                                    <h5>1.1.1.3. Section 1.1.1.3</h5>
                                 </div>
                                 <div class="content">
                                    <p>blah.</p>
                                 </div>
                              </section>
                           </div>
                        </section>
                        <section id="c2s1s1s2" class="section">
                           <div class="section-titlepage">
                              <h4>1.1.2. Section 1.1.2</h4>
                           </div>
                           <div class="content">
                              <p>blah.</p>
                           </div>
                        </section>
                        <section id="c2s1s1s3" class="section">
                           <div class="section-titlepage">
                              <h4>1.1.3. Section 1.1.3</h4>
                           </div>
                           <div class="content">
                              <p>blah.</p>
                           </div>
                        </section>
                     </div>
                  </section>
                  <section id="c2s1s2" class="section">
                     <div class="section-titlepage">
                        <h3>1.2. Section 1.2</h3>
                     </div>
                     <div class="content">
                        <p>blah.</p>
                     </div>
                  </section>
                  <section id="c2s1s3" class="section">
                     <div class="section-titlepage">
                        <h3>1.3. Section 1.3</h3>
                     </div>
                     <div class="content">
                        <p>blah.</p>
                     </div>
                  </section>
               </div>
            </section>
         </div><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><section deltaxml:deltaV2="A" id="c2s1" class="section">
            <div class="section-titlepage">
               <h2>1. Section 1</h2>
            </div>
            <p>blah.</p>
            <section id="c2s1s1" class="section">
               <div class="section-titlepage">
                  <h3>1.1. Section 1.1</h3>
               </div>
               <p>blah.</p>
               <section id="c2s1s1s1" class="section">
                  <div class="section-titlepage">
                     <h4>1.1.1. Section 1.1.1</h4>
                  </div>
                  <p>blah.</p>
                  <section id="c2s1s1s1s1" class="section">
                     <div class="section-titlepage">
                        <h5>1.1.1.1. Section 1.1.1.1</h5>
                     </div>
                     <p>blah.</p>
                     <section id="c2s1s1s1s1s1" class="section">
                        <div class="section-titlepage">
                           <h6>1.1.1.1.1. Section 1.1.1.1.1</h6>
                        </div>
                        <p>blah.</p>
                     </section>
                     <section id="c2s1s1s1s1s2" class="section">
                        <div class="section-titlepage">
                           <h6>1.1.1.1.2. Section 1.1.1.1.2</h6>
                        </div>
                        <p>blah.</p>
                     </section>
                     <section id="c2s1s1s1s1s3" class="section">
                        <div class="section-titlepage">
                           <h6>1.1.1.1.3. Section 1.1.1.1.3</h6>
                        </div>
                        <p>blah.</p>
                     </section>
                  </section>
                  <section id="c2s1s1s1s2" class="section">
                     <div class="section-titlepage">
                        <h5>1.1.1.2. Section 1.1.1.2</h5>
                     </div>
                     <p>blah.</p>
                  </section>
                  <section id="c2s1s1s1s3" class="section">
                     <div class="section-titlepage">
                        <h5>1.1.1.3. Section 1.1.1.3</h5>
                     </div>
                     <p>blah.</p>
                  </section>
               </section>
               <section id="c2s1s1s2" class="section">
                  <div class="section-titlepage">
                     <h4>1.1.2. Section 1.1.2</h4>
                  </div>
                  <p>blah.</p>
               </section>
               <section id="c2s1s1s3" class="section">
                  <div class="section-titlepage">
                     <h4>1.1.3. Section 1.1.3</h4>
                  </div>
                  <p>blah.</p>
               </section>
            </section>
            <section id="c2s1s2" class="section">
               <div class="section-titlepage">
                  <h3>1.2. Section 1.2</h3>
               </div>
               <p>blah.</p>
            </section>
            <section id="c2s1s3" class="section">
               <div class="section-titlepage">
                  <h3>1.3. Section 1.3</h3>
               </div>
               <p>blah.</p>
            </section>
         </section>
      </article>
      <script deltaxml:deltaV2="A=B" src="../../resources/js/prism.js"/>
   </body>
</html>