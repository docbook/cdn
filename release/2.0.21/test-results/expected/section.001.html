<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Article wrapper</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.17" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><div class="lists-of-titles"><div class="toc"><p><b>Table of Contents</b></p><ul class="toc"><li><span><a href="#c2s1">1. Section 1</a></span><ul class="toc"><li><span><a href="#c2s1s1">1.1. Section 1.1</a></span></li><li><span><a href="#c2s1s2">1.2. Section 1.2</a></span></li><li><span><a href="#c2s1s3">1.3. Section 1.3</a></span></li></ul></li></ul></div></div><p>See <a href="#c2s1s1s2" title="Section 1.1.2">Section 1.1.2, “Section 1.1.2”</a>.</p><section id="c2s1" class="section"><div class="section-titlepage"><h2>1. Section 1</h2></div><p>blah.</p><section id="c2s1s1" class="section"><div class="section-titlepage"><h3>1.1. Section 1.1</h3></div><p>blah.</p><section id="c2s1s1s1" class="section"><div class="section-titlepage"><h4>1.1.1. Section 1.1.1</h4></div><p>blah.</p><section id="c2s1s1s1s1" class="section"><div class="section-titlepage"><h5>1.1.1.1. Section 1.1.1.1</h5></div><p>blah.</p><section id="c2s1s1s1s1s1" class="section"><div class="section-titlepage"><h6>1.1.1.1.1. Section 1.1.1.1.1</h6></div><p>blah.</p></section><section id="c2s1s1s1s1s2" class="section"><div class="section-titlepage"><h6>1.1.1.1.2. Section 1.1.1.1.2</h6></div><p>blah.</p></section><section id="c2s1s1s1s1s3" class="section"><div class="section-titlepage"><h6>1.1.1.1.3. Section 1.1.1.1.3</h6></div><p>blah.</p></section></section><section id="c2s1s1s1s2" class="section"><div class="section-titlepage"><h5>1.1.1.2. Section 1.1.1.2</h5></div><p>blah.</p></section><section id="c2s1s1s1s3" class="section"><div class="section-titlepage"><h5>1.1.1.3. Section 1.1.1.3</h5></div><p>blah.</p></section></section><section id="c2s1s1s2" class="section"><div class="section-titlepage"><h4>1.1.2. Section 1.1.2</h4></div><p>blah.</p></section><section id="c2s1s1s3" class="section"><div class="section-titlepage"><h4>1.1.3. Section 1.1.3</h4></div><p>blah.</p></section></section><section id="c2s1s2" class="section"><div class="section-titlepage"><h3>1.2. Section 1.2</h3></div><p>blah.</p></section><section id="c2s1s3" class="section"><div class="section-titlepage"><h3>1.3. Section 1.3</h3></div><p>blah.</p></section></section></article><script src="../../resources/js/prism.js"></script></body></html>